src/LineChart/__tests__/data.spec.ts

```diff
@@ -0,0 +1,121 @@
+import { ScatterData } from "plotly.js";
+import { Data } from "../../utils/namespaces";
+import * as LineChartData from "../utils/data";
+import { LineChartDataHandlerProps } from "../components/LineChartDataHandler";
+import * as dataUtils from "../../utils/data";
+
+describe("LineChart/utils/data", () => {
+    describe("#getData", () => {
+        it("returns chart data", () => {
+            const series: Partial<Data.LineSeriesProps> = {
+                name: "Series 1",
+                xValueAttribute: "xValue",
+                xValueSortAttribute: "createdDate",
+                yValueAttribute: "yValue",
+                lineColor: ""
+            };
+            const props: Partial<LineChartDataHandlerProps> = {
+                devMode: "basic",
+                themeConfigs: {
+                    layout: {},
+                    data: {}
+                } as any
+            };
+            const serie: Data.SeriesData<Data.LineSeriesProps> = {
+                data: [
+                    {
+                        get: jasmine.createSpy("get")
+                    }
+                ] as any,
+                restData: undefined,
+                series : series as Data.LineSeriesProps
+            };
+
+            spyOn(dataUtils, "getSeriesTraces").and.callFake(() => {
+                return { x: [ "Jan", "Feb", "Mar" ], y: [ 4, 5, 6 ] };
+            });
+
+            const data = LineChartData.getData([ serie ], props as LineChartDataHandlerProps);
+
+            expect(data).toEqual([ {
+                connectgaps: true,
+                hoverinfo: "none",
+                hoveron: "points",
+                line: { color: "rgba(5, 149, 219, 1)", shape: undefined },
+                mode: "lines",
+                name: "Series 1",
+                type: "scatter",
+                fill: "none",
+                marker: { },
+                x: [ "Jan", "Feb", "Mar" ],
+                y: [ 4, 5, 6 ],
+                text: "",
+                customdata: [ { get: jasmine.any(Function) } ],
+                series: { name: "Series 1", xValueAttribute: "xValue", xValueSortAttribute: "createdDate", yValueAttribute: "yValue", lineColor: "" },
+                visible: true
+            } ] as any);
+        });
+    });
+
+    describe("#getStackedArea", () => {
+        it("returns stacked area", () => {
```

Comment:
The true stacked area test should take at least two series and return stacked data